### ISSUE TITLE
Accessibility compliance: input element, type of "text", missing an associated label 

### ISSUE THREAD

**Issue by ivanteoh** [eea/eea.facetednavigation#29]
Add a label element that surrounds the control's label. Set the for attribute on the label element to the same value as the idattribute of the control. And/or add a title attribute to the input element. And/or create a label element that contains the inputelement.

For example, the search input text box does not have label close to it, refer to screen shot. 

Refer to [Web accessibility checker 57](http://achecker.ca/checker/suggestion.php?id=57)

![Resources MHCS - Google Chrome_266](https://f.cloud.github.com/assets/900434/359965/cb00f6e0-a177-11e2-816e-33084ef5ca83.png)

Thank you.
